client/my-sites/theme/bundle-settings.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactElement } from 'react';
+
+interface BundleSettings {
+	[ key: string ]: {
+		name: string;
+		badgeIcon: ReactElement;
+	};
+}
+
+const bundleSettings: BundleSettings = {
+	// TODO: Move to a constant.
+	'woo-on-plans': {
+		name: 'WooCommerce',
+		badgeIcon: (
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
The original icon was changed to this one, so it uses the currentColor to set the icon as white, and the "Woo" text is transparent. It will make it easier to create the icons generically for all bundles.